graphql/fetchUserGoalsCount.ts

```diff
@@ -0,0 +1,33 @@
+import { gql } from "@apollo/client";
+
+export const FETCH_USER_GOALS_COUNT = gql`
```

Comment:
this looks like the old version of your fetch user goals count query, you should remove this